omnibus/Gemfile

```diff
@@ -23,7 +13,13 @@ group :development do
   gem 'berkshelf', '>= 7.0'
 
   # Use Test Kitchen with Vagrant for converging the build environment
-  gem 'kitchen-vagrant', '~> 1.3.1'
-  gem 'test-kitchen',    '~> 1.21'
+  gem 'kitchen-vagrant', '>= 1.3.1'
+  gem 'test-kitchen', '>= 1.23'
```

Comment:
This allows us to start using test-kitchen 2.0 which we want for licensing support